### PR TITLE
update test case for 2 processors situation

### PR DESCRIPTION
**Pull request by wangleis**
### Details:
 - *update test case for 2 processors situation*

### Tickets:
 - *109125*

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp [hunk 1]
```diff
@@ -181,19 +181,19 @@ INSTANTIATE_TEST_SUITE_P(smoke_OVClassSetDevicePriorityConfigPropsTest,
                                             ::testing::ValuesIn(multiConfigs)));
 
 const std::vector<ov::AnyMap> configsDeviceProperties = {
-    {ov::device::properties("CPU", ov::num_streams(3))},
-    {ov::device::properties(ov::AnyMap{{"CPU", ov::AnyMap{ov::num_streams(3)}}})}};
+    {ov::device::properties("CPU", ov::num_streams(2))},
```
Comment (riverlijunjie) [inline src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp:184]: How about 1 processor situation?

Comment (wangleis) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp:184]: In current design, CPU inference is always use 1 stream on 1 processor platform. 

This test case is not valid on 1 processor situation.

**Review by riverlijunjie — APPROVED** [review]

**Review by xufang-lisa — APPROVED** [review]

**Review by sunxiaoxia2022 — APPROVED** [review]

**Comment by wangleis** [comment]
build_jenkins

**Comment by wangleis** [comment]
build jenkins

**Review by dmitry-gorokhov — APPROVED** [review]

**Comment by dorloff** [comment]
Can we merge this PR?

**Comment by wangleis** [comment]
@dorloff merged

**Pull request merged**